README.md

```diff
@@ -1,60 +1,85 @@
 CakePHP Sample App on OpenShift
------------------
+===============================
 
-This example will serve an http response of various "machine" stats from the "machine" the CakePHP app is running on to [http://host:8080](http://host:8080).
+This is a quickstart CakePHP application for OpenShift v3 that you can use as a starting point to develop your own application and deploy it on an [OpenShift](https://github.com/openshift/origin) cluster.
+
+If you'd like to install it, follow [these directions](https://github.com/openshift/cakephp-ex/blob/master/README.md#installation).  
 
-### OpenShift setup ###
+The steps in this assume that you have access to an OpenShift deployment; you must have an OpenShift deployment that you have access to in order to deploy this app.
+
+This example will serve an http response of various "machine" stats from the "machine" the CakePHP app is running on to [http://host:8080](http://host:8080).
```

Comment:
> serve an HTTP response :us:

or

> In order to access the default CakePHP welcome page* with application stats, go to the `app/Views/Layouts/` directory, remove the `default.ctp` file and rename `default.ctp.default` to `default.ctp`.

\* as they call it in their [docs](http://book.cakephp.org/2.0/en/getting-started.html#cakephp-database-configuration).